algorithms/utils/action_distributions.py

```diff
@@ -193,6 +203,10 @@ def kl_divergence(self, other):
         kl = kls.sum(dim=1)
         return kl
 
+    def symmetric_kl_with_uniform_prior(self):
+        kls = [d.symmetric_kl_with_uniform_prior() for d in self.distributions]
```

Comment:
Have you tested this? I think this is expected to return a PyTorch scalar, not a list of scalars.
See function kl_divergence()

Reply:
If I add  `kl = action_distribution.kl_divergence(action_distribution)` in `test_tuple_distribution` of `test_action_distributions.py` it returns a tensor of size 42 (obviously it'll be all zeros since it is the KL divergence between the same distributions, just added it for the test). Is that correct?